.github/drafts/16506/16506_slider_widget_landing_draft.md

```diff
@@ -0,0 +1,54 @@
+# Sliders
+Sliders are useful widgets that allow you to capture user input related to a range of data, and offer visual feedback to users as they select values. Appsmith has three varieties of these widgets: Category, Numerical, and Range sliders.
+
+---
+## Add to Canvas
+In the left navigation pane, navigate to PAGES —> Select the "Widget" Tab —> Write “Slider". Choose your desired slider variant from the three types that are available.
+
+To add your desired slider widget to your canvas, click and drag it from the widget pane onto the canvas.
+
+Now that you have a slider widget added to the canvas, you can move it anywhere on the canvas by simply dragging it around.
+
+## Usage
+
+### Category Slider
+The Category Slider widget can be useful for situations where users must provide feedback along a scale that isn't numerical. Surveys, for example, often have questions which ask respondents to rate how much they agree or disagree with certain statements (that is, Strongly agree/ agree/ disagree/ strongly disagree).
+
+![](as_category.png)
+
+{% content-ref url="category_slider.md" %} category_slider.md {% endcontent-ref %}
+
+### Number Slider
```

Comment:
Please check https://mantine.dev/core/slider/ , for more examples and use-cases. I don't think we should include the page number as a use-case for our slider.